### PR TITLE
[sdg] multiple un ux fixes

### PR DESCRIPTION
**Pull request by dwnoble**
* Added sidebar tooltips
* Added page title "UNSD - Data Commons for the SDGs" (UN can refine this later)
* Set search page background color to gray from https://datcom-un.ue.r.appspot.com/
* Added datacommons info text at the page footer
* Fixed 12.3.1b, 14.1.1a, 14.1.1b infographics so they display on the sections 12.3.1 and 14.1.1 respectively

## Tooltips
un-ux-sidebar-tooltips

## Footer info text
Screenshot 2023-09-12 at 3 10 21 PM

### CODE REVIEW

File: experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/shared/components.tsx [hunk 1]
```diff
@@ -548,7 +558,11 @@ export const Footnotes: React.FC = () => {
       <FootnoteDivider></FootnoteDivider>
       <Footnote>
         <StyledMarker>*</StyledMarker>
-        {MAP_DISCLAIMER_TEXT}
+        <div>{MAP_DISCLAIMER_TEXT}</div>
+      </Footnote>
+      <Footnote>
+        <StyledMarker>*</StyledMarker>
```
Comment (juliawu) [inline experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/shared/components.tsx:564]: Perhaps we should update or remove the marker for the info text? Right now the `*` for the map disclaimer text works like a citation, each map chart has an asterisk to point to the disclaimer. Looks weird to have two `*` references.

**Review by juliawu — COMMENTED** [review]
Nice job with the tooltip! Just a quick comment.

File: experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/shared/components.tsx (hunk 1 above)
Comment (dwnoble) [reply experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/shared/components.tsx:564]: Done! updated screenshot
Screenshot 2023-09-12 at 4 55 19 PM

**Review by juliawu — APPROVED** [review]
Thanks for updating!

**Pull request merged**